### DISCUSSION TITLE
pre-caching an SSR page

### DISCUSSION THREAD

**Discussion by drewda** [nuxt/nuxt#26705] [Questions] [answered]
We're running Nuxt 3 in SSR mode. There's one page that has a slow GraphQL query (using Apollo). The data rarely changes, so we're thinking about ways to cache/persist this page and rebuild it occasionally.

We could generate it statically when we deploy the site, but that may be too infrequent.

Are there any ways that Nuxt 3 can cache an SSR page for a specific amount of time -- ideally for some hours or a day?

Thank you for any pointers you all can offer.

**Comment by xjccc** [comment, marked as answer]
look for this
[routeRules](https://nitro.unjs.io/config#routerules)

**Reply by drewda** [reply]
Thanks for your help. This does look potentially useful: https://nitro.unjs.io/guide/cache
